GraphClient.go

```diff
@@ -304,6 +304,32 @@ func (g *GraphClient) CreateUser(userInput User, opts ...CreateQueryOption) (Use
 	return user, err
 }
 
+// SendMail sends a message.
+//
+// See https://docs.microsoft.com/en-us/graph/api/user-sendmail
+func (g *GraphClient) SendMail(userID string, message Message, saveToSentItems bool) error {
```

Comment:
This is great, I saw that in the previous commit 9905cba7bacb01d32c7e08a5e401f97633c39377 you had the SendMail function in the `User` struct. Please move it there again, and write a wrapper function here, something like:
````go
func (g *GraphClient) SendMail(userID string, message Message, saveToSentItems bool) error {
  user := User{ ID: userID, graphClient: g}
  return user.SendMail(message, saveToSentItems)
}
````

Please also add a hint to the function comment / description that this is just a wrapper referencing to the `User` struct